gensim/models/word2vec.py

```diff
@@ -1181,33 +1189,103 @@ def intersect_word2vec_format(self, fname, lockf=0.0, binary=False, encoding='ut
         logger.info("merged %d vectors into %s matrix from %s" % (overlap_count, self.wv.syn0.shape, fname))
 
     def most_similar(self, positive=[], negative=[], topn=10, restrict_vocab=None, indexer=None):
+        """
+        Please refer to the documentation for
+        `gensim.models.KeyedVectors.most_similar`
+        This is just a forwarding function.
+        In the future please use the `gensim.models.KeyedVectors` instance in wv
```

Comment:
Hmm, The strange line breaks were added by me in the [fix pep8](https://github.com/RaRe-Technologies/gensim/pull/1251/commits/73396652986654e19e1ad6f415f00ce5324f5c73) commit! I'm new to following pep8 all the time, do you think I overdid it there?

Is a depreciated warning to be added to these methods? Will they become unavailable in a later version? If yes, then this should surely be changed asap and I'll send a new PR with the same.